html/html.go

```diff
@@ -37,7 +39,16 @@ func (p *Page) Render(w http.ResponseWriter) error {
 		return err
 	}
 
-	return tmpl.ExecuteTemplate(w, "application.html", p)
+	var buffer bytes.Buffer
+	err = tmpl.ExecuteTemplate(&buffer, "application.html", p)
```

Comment:
I tried with `NewWriter` as specified in https://github.com/yosssi/gohtml but nothing came out of it. Can you take a look @athoune ?

Reply:
I can't do better, `gothtml.Writer` behavior is weird.